packages/core-modules/test/contracts/modules/CoreElectionModule.test.js

```diff
@@ -238,44 +245,63 @@ describe('CoreElectionModule', () => {
     });
 
     it('reverts when trying to elect no candidates', async () => {
-      await assertRevert(CoreElectionModule.connect(user).elect([]), 'InvalidCandidatesCount');
+      await assertRevert(
+        CoreElectionModule.connect(user).elect([], []),
+        'InvalidCandidatesAddresses'
+      );
     });
 
     it('reverts when trying to elect more candidates than nominees', async () => {
       await assertRevert(
-        CoreElectionModule.connect(user).elect([
-          user.address,
-          owner.address,
-          candidates[0].address,
-          candidates[1].address,
-          candidates[2].address,
-          candidates[3].address,
-          candidates[4].address,
-        ]),
-        'InvalidCandidatesCount'
+        CoreElectionModule.connect(user).elect(
+          [
+            user.address,
+            owner.address,
+            candidates[0].address,
+            candidates[1].address,
+            candidates[2].address,
+            candidates[3].address,
+            candidates[4].address,
+          ],
+          [0, 1, 2, 3, 4, 5, 6, 7]
+        ),
+        'InvalidCandidatesAddresses'
```

Comment:
See my comment about errors above. With that, this test would be much easier to read with `TooManyCandidates`.

Reply:
Done: https://github.com/Synthetixio/synthetix-v3/pull/572/files#diff-69650d386e562b4dac992709531657fe63fedd176cde00f236b7b6271333c70eR152-R154